packages/storybook8/stories/Composites/CallComposite/pstn/Docs.mdx

```diff
@@ -10,9 +10,10 @@ Azure Communication Services call service. The UI Library renders a dialpad that
 features such as being put on hold and dialing in and removing users from an existing call.
 
 More specifically, the following features are supported:
+http://localhost:6006/iframe.html?viewMode=docs&id=components-dialpad--docs&args=#dialer-mode
```

Comment:
This is pointing to localhost, I think this was actually unintended 
```suggestion

```

Reply:
interesting I might have not pushed the update since this is what I am seeing
![Uploading image.png…]()

Reply:
should be removed now

![image](https://github.com/user-attachments/assets/5631f952-8d9b-4c40-bf36-fbc8ee38d310)